doc/Language/control.pod6

```diff
@@ -927,6 +951,25 @@ for @x -> $x {
 
 prints "12".
 
+If the C<LAST> L<phaser|/language/phasers> is present, it runs:
```

Comment:
It runs when? No problem if you repeat part of the statement before the example.
Also, I'm not sure it belongs here. As long as you have introduced a new section, wouldn't it be better if you put it there?